pkg/api/types.go

```diff
@@ -1345,6 +1345,16 @@ type NodeSpec struct {
 	Unschedulable bool `json:"unschedulable,omitempty"`
 }
 
+// KubeletPorts lists ports opened by Kubelet
+type KubeletPorts struct {
+	// KubeletPort on which Kubelet is listening
+	KubeletPort uint `json:kubeletPort`
+	// KubeletReadOnlyPorty only port open on Kubelet
+	KubeletReadOnlyPorty uint `json:kubeletReadOnlyPort`
```

Comment:
What still uses the read-only port? When I created it (for heapster), @vishh told me that heapster would move to use the secured port and that we could quickly deprecated the RO port. I know that monit was using it as well -- does supervisord need it or can it make an https connection?